src/coreclr/src/jit/codegencommon.cpp

```diff
@@ -3991,40 +3991,40 @@ void CodeGen::genFnPrologCalleeRegArgs(regNumber xtraReg, bool* pXtraRegClobbere
 
         if (doingFloat)
         {
-#if defined(FEATURE_HFA) || defined(UNIX_AMD64_ABI)
-            insCopy = ins_Copy(TYP_DOUBLE);
-            // Compute xtraReg here when we have a float argument
-            assert(xtraReg == REG_NA);
+#ifdef UNIX_AMD64_ABI
+            if (true)
+#else  // !UNIX_AMD64_ABI
+            if (GlobalJitOptions::compFeatureHfa)
+#endif // UNIX_AMD64_ABI
```

Comment:
Couldn't this just be:
```
#if !defined(UNIX_AMD64_ABI)
            if (GlobalJitOptions::compFeatureHfa)
#endif // !UNIX_AMD64_ABI
```

Reply:
Yes.

---
In reply to: [510492047](https://github.com/dotnet/runtime/pull/43706#discussion_r510492047) [](ancestors = 510492047)